packages/pulpcore/python-pulp-container/python-pulp-container.spec

```diff
@@ -20,14 +20,14 @@ BuildRequires:  python3-setuptools
 %package -n     python3-%{pypi_name}
 Summary:        %{summary}
 %{?python_provide:%python_provide python3-%{pypi_name}}
-Requires:       python3-pulpcore < 3.1
-Requires:       python3-pulpcore >= 3.0.0
-Requires:       python3-pyjwkest >= 1.4.0
-Conflicts:      python3-pyjwkest >= 1.5
 Requires:       python3-ecdsa >= 0.13.2
 Conflicts:      python3-ecdsa >= 0.14
-Requires:       python3-jwt >= 1.7.1
-Conflicts:      python3-jwt >= 1.8
+Requires:       python3-pulpcore < 3.3
+Requires:       python3-pulpcore >= 3.2.0
+Requires:       python3-pyjwkest >= 1.4.0
+Conflicts:      python3-pyjwkest >= 1.5
+Requires:       python3-pyjwt >= 1.7.1
```

Comment:
the change here (from jwt to pyjwt) is actually correct, our package provides both names, but pyjwt is the correcter one, no idea why it was "jwt" before. :+1: